06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable no-undef */
+
+const invalidKeys = require('./invalidKeys.json'); // bad content
+const validBook = require('./validBook.json'); // good content
+const fileTextString = require('./fileTextString.json'); // text objects
```

Comment:
Are you supossed to use this for any of your test specifications? because 'fileTextString' is assigned a value but never used.